tests/framework/extensions/clusters/clusters.go

```diff
@@ -115,13 +115,32 @@ func CheckServiceAccountTokenSecret(client *rancher.Client, clusterName string)
 		logrus.Infof("serviceAccountTokenSecret in this cluster is: %s", cluster.ServiceAccountTokenSecret)
 		return true, nil
 	} else {
-		logrus.Infof("serviceAccountTokenSecret does not exist in this cluster!")
+		logrus.Infof("warning: serviceAccountTokenSecret does not exist in this cluster!")
 		return false, nil
 	}
 }
 
+// Verify if PSACT is enabled or not in the cluster.
+func CheckPSACT(client *rancher.Client, clusterName string) error {
+	clusterID, err := GetClusterIDByName(client, clusterName)
+	if err != nil {
+		return err
+	}
```

Comment:
Do we want to add logging in the event err != nil?  like something saying, "failed to locate cluster and grab clusterID"?

That's the only improvement I propose.  But if redundant w/ the error that is already being returned, I could see this not really being applicable also.  Just calling it out to see the teams thoughts

Everything else LGTM. great work!

Reply:
Usually we just leave the error checking as presented. Unsure if that is by best practice, but we'd have to make those changes throughout the framework since this happens a lot just off of thinking more on it.